README_API.md

```diff
@@ -0,0 +1,338 @@
+Based Subscriber Identification is used (see above).
+
+|<!-- -->| <!-- -->|
+|:---|:---|
+|**Request Method**|POST|||
+|**Request URI**|https://app.aquto.com/api/datarewards/api/reward|
+
+|Query Parameter|Type|Required|Description|
+|:-------------:|:-------------:|:-----:|:-----|
+|**campaignId**|	Integer	|Yes|	The Aquto portal generated campaign identifier of the sponsor’s campaign.
+|**advertiserId**|	Integer|	Conditional|	The sponsor’s Aquto portal generated advertiser identifier. Required if userIdentifer is passed.
+|**userIdentifier**|	String|	Conditional|	The sponsor’s user identifier. Only required if Network Based Subscriber Identification is used and phoneNumber is not provided.
+|**phoneNumber**|	String|	Conditional	|The phone number of the subscriber. Only required if the userIdentifier is not provided.
+|**couponId**|	Integer|	Yes|	Coupon identifier returned in the eligibility response. 
+|**transactionId**|	String|	Conditional|	Optional sponsor transactionId that will be logged with the transaction. Used for reconciliation purposes. 
+
+
+|Response Object|Type|Description||
+|:-------------:|:-------------:|:-----:|:-----|
+| **Response**|Reward Response|See Reward Response definition.||
+| **Reward Response**|**JSON Field**|**Type**|**Description**|
+|<!-- -->|successful|Boolean|True if the transaction was successful.|
+|<!-- -->|status|	String| **success** - Conversion succeeded |
+|<!-- -->|<!-- -->|	<!-- -->| **unabletoidentify** - Unable to identify the user |
+|<!-- -->|<!-- -->| <!-- -->| **ineligible** - Subscriber not eligible or campaign is inactive |
+|<!-- -->|<!-- -->|	<!-- -->| **unabletoconvert** - Unable to initiate conversion |
+|<!-- -->|<!-- -->|	<!-- -->| **generalerror** -  System error |
+|<!-- -->|operatorCode|	String|	The Aquto portal operator code. See the appendix for a list of supported operators and codes.|
+|<!-- -->|rewardAmountMB|	Integer	|The amount of the coupon in MB.|
+|<!-- -->|transactionId|	String|	The Aquto transactionId of the transaction. Can be stored by the sponsor for future reconciliation.|
+|<!-- -->|price	|Double|	The price of the reward to the sponsor in the sponsor’s currency.|
+|<!-- -->|currency|	String|	The ISO-4217 currency code of the sponsor’s currency.|
+|<!-- -->|networkPrice|	Double|	The price of the reward in the network/operator’s currency.|
+|<!-- -->|networkCurrency|	String|	The ISO-4217 currency code of the operator’s currency.|
+
+
+
+***Example Request***
+```
+POST /api/datarewards/api/reward
+```
+
+```json
+{
+ "campaignId": 123,
+ "phoneNumber": "15554441212"
+ "couponId": 5
+}
+```
+
+***Example Response***
+
+```json
+{
+ "meta": {
+  "code": 200
+ },
+ "response": {
+  "successful": true,
+  "status": "success",
+  "operatorCode": "opabc",
+  "rewardAmountMB": 10,
+  "transactionId": "cl_12345_123"
+  "price": 0.35,
+  "currency": "USD",
+  "networkPrice": 7.02,
+  "networkCurrency": "MXN"
+ }
+}
+```
+
+Appendix A - Supported Operator Codes
```

Comment:
Can you remove this section please so we don't need to maintain it when things change.